Makefile.in

```diff
@@ -36,6 +36,8 @@ targetdir = $(top_builddir)
 INSTALL_FLAGS = install
 UNINSTALL_FLAGS = uninstall
 
+DMTCP_ROOT ?=$(top_builddir)/dmtcp
```

Comment:
Please add space after `?=`:
  `DMTCP_ROOT ?= $(top_builddir)/dmtcp`

Also, before this line, please add the comment:
  `# If 'DMTCP_ROOT' env. var. is defined, use that instead of dmtcp submodule.`

Reply:
Will do.